src/js/components/stock-movement/inbound/StockMovementInboundTable.jsx

```diff
@@ -178,14 +175,13 @@ const StockMovementInboundTable = ({
       sortable: false,
       Cell: (row) => {
         const {
-          isReturn, id, order, description, name,
+          id, description, name,
         } = row.original;
-        const stockMovementId = isReturn ? order?.id : id;
         return (
           <TableCell
             {...row}
             tooltip
-            link={STOCK_MOVEMENT_URL.show(stockMovementId)}
+            link={STOCK_MOVEMENT_URL.show(id)}
```

Comment:
can you explain how this fixes the underlying issue? It looks like this is just changing links in the table to not link to the order anymore. Isn't the real issue that when "rollback" is clicked while on the inbound return edit page it doesn't check if there's sufficient quantity to perform the rollback?